packages/headless/jest.config.js

```diff
@@ -1,6 +1,4 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  silent: true,
-  verbose: true,
```

Comment:
Removed these options because:
- `silent` is not preventing logs to the console.
- `verbose` causes the terminal to scroll for a very long time (we have ~2k unit tests). I haven't personally seen a benefit in listing passing unit tests.